test/chat.test.js

```diff
@@ -239,7 +240,8 @@ describe('Test Suite for chat', () => {
                     {
                         users_id: expect.any(Number),
                         emoji: expect.any(String)
-                    }]
+                    }],
+                    recommended: expect.arrayContaining([])
```

Comment:
Forgot to add this earlier, but you might consider adding a test later (once you've figured out more of the NLP API details) where you verify it returns some non-empty expected data.